packages/react-native-web/src/modules/createDOMProps/index.js

```diff
@@ -187,8 +187,12 @@ const createDOMProps = (component, props, styleResolver) => {
   // https://mathiasbynens.github.io/rel-noopener/
   // Note: using "noreferrer" doesn't impact referrer tracking for https
   // transfers (i.e., from https to https).
-  if (component === 'a' && domProps.target === '_blank') {
-    domProps.rel = `${domProps.rel || ''} noopener noreferrer`;
+  // Note: Specifying rel="opener" will explicitly bypass this safeguard.
+  if (component === 'a' && domProps.target === '_blank' && domProps.rel !== 'opener') {
+    const existingRel = domProps.rel ? domProps.rel.split(' ') : [];
+    // Ensure that we don't end up with duplicates.
+    const newRel = new Set([...existingRel, 'noopener', 'noreferrer']);
```

Comment:
Creating a Set, array, then flattening it is unnecessary overhead on a hot path

Reply:
OK, I guess I thought this was the simplest way to prevent the duplicates that I saw in my DOM when I was using this. I don't know that it's a bug necessarily to have duplicates but I'm not sure that it's well defined in the spec what should happen if multiple conflicting or duplicates are in the `rel` attribute. I'll revert this change for a more performant one.